dbt_coves/tasks/generate/sources.py

```diff
@@ -222,9 +223,22 @@ def generate_sources(self, rels):
 
     def generate_model(self, relation, destination, options):
         destination.parent.mkdir(parents=True, exist_ok=True)
-        columns = self.adapter.get_columns_in_relation(relation)
-        nested_field_type = NESTED_FIELD_TYPES.get(self.adapter.__class__.__name__)
-        nested = [col.name.lower() for col in columns if col.dtype == nested_field_type]
+        metadata = self.get_metadata(self.get_config_value("metadata"))
+
+        if metadata:
```

Comment:
@LeonardoPelanda the issue here is that if the metadata file contains metadata for only a few fields, then the other table fields will not be generated. I recommend loading the column data first, and then adding the metadata if exists. If a field is not in the metadata file, it sill should be generated.